src/sentry/tasks/sentry_apps.py

```diff
@@ -116,10 +117,23 @@ def send_alert_event(event, rule, sentry_app_id):
 def _process_resource_change(action, sender, instance_id, retryer=None, *args, **kwargs):
     # The class is serialized as a string when enqueueing the class.
     model = TYPES[sender]
-
-    # Some resources are named differently than their model. eg. Group vs
-    # Issue. Looks up the human name for the model. Defaults to the model name.
-    name = RESOURCE_RENAMES.get(model.__name__, model.__name__.lower())
+    # The Event model has different hooks for the differenct types. The sender
+    # determines which type eg. Error and therefore the 'name' eg. error
+    if model.__name__ == 'Event':
```

Comment:
Pretty sure also `SnubaEvent` will end up here. Can we find a better way to detect this?